### ISSUE TITLE
Target Segmentation Mask would be required for accuracy

### ISSUE THREAD

**Issue by tryontester** [Yagami360/virtual-try-on_exercises_pytorch#4]
One more additional step of creation of target segmentation mask would be required before TOM. This May solve the problem of half sleeve to full sleeve etc

**Comment by Yagami360** [comment]
Yes, I also think that adding a module to generate person perspective images like MG-VTON will lead to improvement of the model (especially half sleeve to full sleeve).

**Comment by tryontester** [comment]
If you check the paper of sievenet they have additional step of conditional target segmentation mask creation . This layer is trained on warped clothing item. Then the final unet is trained with target segmentation mask.
